### PR TITLE
Update TensorFlow deployment

### PR DESCRIPTION
**Pull request by microcosem**
Changes submitted by Eta. Thanks!

Relates to coreweave/docs#123

[See Slack for further context](https://coreweave.slack.com/archives/C03L3KJ7708/p1679627730034709?thread_ts=1679005771.812899&cid=C03L3KJ7708).

### CODE REVIEW

File: tensorflow-jupyter/tensorflow-deployment.yaml [hunk 1]
```diff
@@ -60,14 +60,14 @@ spec:
       # Read more about affinity at: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
       affinity:
         nodeAffinity:
-          # This will REQUIRE the Pod to be run on a system with a Pascal GPU
+          # This will REQUIRE the Pod to be run on a system with an NVIDIA A40 GPU
```
Comment (Eta0) [inline tensorflow-jupyter/tensorflow-deployment.yaml:63]: I added a commit to update this comment to align with the change in GPU selection

File: tensorflow-jupyter/tensorflow-deployment.yaml [hunk 2]
```diff
@@ -78,16 +78,16 @@ spec:
           # If systems with i5 / i9 / Xeon CPUs and/or 1G ethernet are not available to fulfill the requested resources, the Pods
           # will be scheduled on higher end systems.
           preferredDuringSchedulingIgnoredDuringExecution:
```
Comment (Eta0) [inline tensorflow-jupyter/tensorflow-deployment.yaml:80]: Noting that this section and its explanatory comment above will need to be updated later to match the CPU types we currently have available. Right now it's just commented out.

File: tensorflow-jupyter/tensorflow-deployment.yaml [hunk 3]
```diff
@@ -60,14 +60,14 @@ spec:
...
+          # This will REQUIRE the Pod to be run on a system with an NVIDIA A40 GPU
           requiredDuringSchedulingIgnoredDuringExecution:
             nodeSelectorTerms:
             - matchExpressions:
               - key: gpu.nvidia.com/class
                 operator: In
                 values:
-                  - NV_Pascal
+                  - A40
               - key: failure-domain.beta.kubernetes.io/region
```
Comment (Eta0) [inline tensorflow-jupyter/tensorflow-deployment.yaml:71]: We should also update keys like these to be up-to-date when we can (e.g. `failure-domain.beta.kubernetes.io/region` → `topology.kubernetes.io/region`)

**Review by Eta0 — APPROVED** [review]
Left some comments on further touch-ups I noticed that this file needs later on, but it should be all good to merge to get it functioning for now.

**Comment by microcosem** [comment]
Just added a commit for changing ORD1 to LGA1 as per @ChachiTheGhost 's suggestion.

**Review by ChachiTheGhost — APPROVED** [review]

**Pull request merged**